REFERENCE.md

```diff
@@ -249,13 +249,14 @@ Some additional notes about the behavior of boosted regression forests:
 
 ### Cluster-Robust Estimation
 
-For accurate predictions and variance estimates, it can be important to take into account for natural clusters in the data, as might occur if a dataset contains examples taken from the same household or small town. GRF provides support for cluster-robust forests by accounting for clusters in the subsampling process. To use this feature, the forest must be trained with the relevant cluster information by specifying the `clusters` and optionally `samples_per_cluster` parameters. Then, all subsequent calls to `predict` with will take clusters into account, including when estimating the variance of forest predictions.
+For accurate predictions and variance estimates, it can be important to take into account for natural clusters in the data, as might occur if a dataset contains examples taken from the same household or small town. GRF provides support for cluster-robust forests by accounting for clusters in the subsampling process. To use this feature, the forest must be trained with the relevant cluster information by specifying the `clusters` and optionally the `clusters.subsample` parameter. Then, all subsequent calls to `predict` will take clusters into account, including when estimating the variance of forest predictions.
 
-When clustering is enabled during training, all subsampling procedures operate on entire clusters as opposed to individual examples. Then, to determine the examples used for performing splitting and populating the leaves, `samples_per_cluster` examples are drawn from the selected clusters. By default, `sample_per_cluster` is equal to the size of the smallest cluster. Concretely, the cluster-robust training procedure proceeds as follows:
+When clustering is enabled during training, all subsampling procedures operate on entire clusters as opposed to individual examples. Then, to determine the examples used for performing splitting and populating the leaves, *Nc* examples are drawn from the selected clusters. By default, *Nc* is all the observations in the clusters. If `clusters.subsample = TRUE` *Nc* is equal to the size of the smallest cluster.
```

Comment:
What is the *Nc* notation style? Is this aligned with well-established R conventions? If not, I'd prefer just writing out `samples_per_cluster` everywhere to avoid the possibility of confusion.